bindings/matrix-sdk-ffi/src/event.rs

```diff
@@ -126,6 +126,8 @@ pub enum MessageLikeEventContent {
     KeyVerificationKey,
     KeyVerificationMac,
     KeyVerificationDone,
+    PollStart,
+    PollEnd,
```

Comment:
I'm struggling to figure out what this enum is actually _for_. Is it just notifications?

I think it can probably be dropped from this PR and reintroduced later, though...

Reply:
Ah, I thought it was for sending these events, but that's not right. Might be only notifications, yeah.